validations/university.validation.js

```diff
@@ -0,0 +1,12 @@
+const { body, param, header } = require("express-validator");
+
+exports.UniversityCreationValidation = [
+    body("name").trim().notEmpty().withMessage("University name is required"),
+    body("pobox").trim().optional(),
+    body("address").trim().optional(),
+    body("country").trim().optional(),
```

Comment:
for all your instances of `optional()` use this [Link](https://stackoverflow.com/questions/24681971/express-validator-how-to-allow-optional-fields) to see the correct usage of it